lib/gym/package_command_generator.rb

```diff
@@ -32,6 +38,11 @@ def pipe
         [""]
       end
 
+      def postfix
+        # Remove the patched PackageApplication file after the export is finished
+        ["; rm -rf #{@patched_package_application}"]
```

Comment:
| Why -r ? It's a file. This is risky if someone starts playing with the.
That's true. We can remove it.

| Why ; ? Maybe && so that we preserve the exit status of the command.

I want to delete any temp file besides the outcome of the previous command.

I have tried the ruby mktemp but besides calling it in a block I haven't found a nice way to delete the temp directory when the application is interrupted or finished.